images/all-spark-notebook/README.md

```diff
@@ -1,11 +1,5 @@
 # Jupyter Notebook Python, R, Spark Stack
 
-> **Images hosted on Docker Hub are no longer updated. Please, use [quay.io image](https://quay.io/repository/jupyter/all-spark-notebook)**
```

Comment:
After I merge this PR:

1. Quay.io descriptions will update automatically
2. Docker Hub descriptions will stay the same (so there will be information, that the images are outdated)